drivers/tee/optee/optee_private.h

```diff
@@ -137,7 +149,7 @@ int optee_supp_recv(struct tee_context *ctx, u32 *func, u32 *num_params,
 int optee_supp_send(struct tee_context *ctx, u32 ret, u32 num_params,
 		    struct tee_param *param);
 
-u32 optee_do_call_with_arg(struct tee_context *ctx, phys_addr_t parg);
+u32 optee_do_call_with_arg(struct tee_device *teedev, phys_addr_t parg);
```

Comment:
Yeah, I agree. I'll look on how to get it back. AFAIR, there was some reason for that change.